### ISSUE TITLE
is the master branch not working for anyone else but me?

### ISSUE THREAD

**Issue by warren-bank** [nexusdev/dapple-quicktest#4]
Hi all. After reading the [blog post](https://blog.dapphub.com/saving-time-and-sanity-using-the-next-generation-of-dapple-testing-tools/) that describes this tool, I was eager to give it a test drive.

* installed rust
  * had to use rust v1.16.0 because the current version (v1.17.0) won't successfully compile `ethrun`
* installed `ethrun`
* installed `dapple-quicktest`

```bash
# simplified instructions for brevity..
mkdir ~/_temp
cd ~/_temp

wget -c --no-check-certificate "https://static.rust-lang.org/dist/rust-1.16.0-x86_64-unknown-linux-gnu.tar.gz"
tar xvfz rust-1.16.0-x86_64-unknown-linux-gnu.tar.gz
cd rust-1.16.0-x86_64-unknown-linux-gnu
sudo ./install.sh

git clone https://github.com/dapphub/ethrun.git
sudo make link -C ethrun

git clone https://github.com/nexusdev/dapple-quicktest.git
sudo make link -C dapple-quicktest
```

* made a test project:

```bash
mkdir ~/_temp/my_project
cd ~/_temp/my_project
dapple init
touch ./src/FooTest.sol
leafpad ./src/FooTest.sol
# copied the very first test contract from the blog post (ie: FooTest)
# saved
```

* here is my output:

```bash
$ dapple test
Testing...

FooTest
  test foo
  Error: VM Exception while executing transaction: out of gas

Failing Tests
  FooTest: test foo
  
Summary
  Failed 1 out of 1 tests.
```

```bash
$ dapple-quicktest
Building...
Trying FooTest...
.

1 tests passed, 0 failed
Took 0.254s
```

I'd really appreciate it if anybody could please point out what (if anything?) that I'm I doing wrong. Thanks :)

**Comment by warren-bank** [comment]
PS)
* this dev environment is Lubuntu 16.04.2 LTS (64-bit)
* it's running as a VirtualBox guest (on a Fedora 23 host OS.. the LXDE spin)
* the guest image is freshly installed, and all updates have been applied
* a few packages were added before rust (ex: node, npm, git, atom, chromium, ...)
* neither `rust`, `ethrun`, nor `dapple-quicktest` displayed any errors during installation/compilation

**Comment by dbrock** [comment]
Thanks for the report. This repo is deprecated and we should probably just delete it.

This program has been merged into https://github.com/dapphub/dapp (and is now invoked as `dapp test`).

Maybe we should update the blog post you linked to as well.

**Comment by warren-bank** [comment]
so much for living on the cutting edge :)

that's great news! thanks.

**Comment by warren-bank** [comment]
this issue was ironed out in `dapp`.

quick closing summary:
* ethrun-0.2.0 and ethrun-0.2.1 had an off-by-1 bug
* the error was fixed in ethrun-0.2.2
* after updating `ethrun` to v0.2.2 (or higher), both `dapp test` and `dapple-quicktest` run properly
